source/rust_verify/src/driver.rs

```diff
@@ -243,20 +250,17 @@ impl<'a> VerusExterns<'a> {
 }
 
 pub fn run<F>(
-    mut verifier: Verifier,
+    verifier: Verifier,
     mut rustc_args: Vec<String>,
+    verus_root: Option<VerusRoot>,
     file_loader: F,
     build_test_mode: bool,
 ) -> (Verifier, Stats, Result<(), ()>)
 where
     F: 'static + rustc_span::source_map::FileLoader + Send + Sync + Clone,
 {
     if !build_test_mode {
-        if let Some(VerusRoot { path: verusroot, in_vargo }) = find_verusroot() {
-            if !verifier.args.no_vstd {
-                let vstd = verusroot.join("vstd.vir").to_str().unwrap().to_string();
-                verifier.args.import.push((format!("vstd"), vstd));
-            }
```

Comment:
I think this removal is an unintentional side-effect of resolving a conflict. Is that correct?